bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -57,10 +58,10 @@ spec:
 
           function quit {
               # Save the flows
-              echo "info: Saving flows ..." 2>&1
+              echo "$TIMESTAMP info: Saving flows ..." 2>&1
               bridges=$(ovs-vsctl -- --real list-br)
               TMPDIR=/var/run/openvswitch /usr/share/openvswitch/scripts/ovs-save save-flows $bridges > /var/run/openvswitch/flows.sh
-              echo "info: Saved flows" 2>&1
+              echo "$TIMESTAMP info: Saved flows" 2>&1
```

Comment:
This will print the same timestamp before and after...

Reply:
good catch. will update.